layers/+lang/asciidoc/README.org

```diff
@@ -22,6 +23,9 @@ To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =asciidoc= to the existing =dotspacemacs-configuration-layers= list in this
 file.
 
+** Mixed pitched support
```

Comment:
Please also add a warning/note about the interaction with company.
See `Interaction with company` section on its repository.